docs/templates/eia930_child.rst.jinja

```diff
@@ -128,6 +128,40 @@ it includes many irregularities including oulying and missing values, which they
 to manage in their aggregated (daily or nationwide) data, but which are present in the
 original reported data.
 
+Changes in energy source granularity over time
+----------------------------------------------
+In the Q1 2025 data release, the wind, solar, and hydro power energy source columns
+were removed from the raw dataset, and new columns for multiple different kinds of
+wind, solar, hydro power, and energy storage were added. The change takes effect
+starting in 2024half2. Analyses that cross this temporal boundary will need to
+aggregate in order to compare e.g. 2025 solar with 2023 solar.
+
+.. list-table::
+   :widths: 20 20
+   :header-rows: 1
+
+   * - Removed in Q1 2025 release
+     - Added in Q1 2025 release
+   * -
+     - ``battery_storage``
+   * -
+     - ``geothermal``
+   * - ``hydro``
+     - ``hydro_excluding_pumped_storage``
+   * -
+     - ``other_energy_storage``
+   * -
+     - ``pumped_storage``
```

Comment:
this is a nit but i might add the `pumped_storage` in with the Added for `hyrdo`.